repo2docker/contentproviders/figshare.py

```diff
@@ -84,3 +96,8 @@ def fetch(self, spec, output_dir, yield_output=False):
     def content_id(self):
         """The Figshare article ID"""
         return self.article_id
+
+    @property
+    def content_version(self):
```

Comment:
The `content_id` is part of the contract between the rest of repo2docker and content providers as the (opaque to the user) value that can be used to identify if the content is the same or not. So we should include the version there instead of creating a new attribute. Put a different way: there can be a new attribute in this content provider if it wants to organise itself that way however for talking to the rest of r2d the `content_id` is what counts.

Reply:
I see - so the `content_id` is more outward facing, and internally I can keep the `article_version`, right?

Reply:
Fixed now